packages/grid_client/scripts/renting.ts

```diff
@@ -1,29 +1,100 @@
+/*
+# Renting Nodes Test Script.
+
+ This script is intended for testing the process of renting nodes. It includes functions to
+  1. reserve
+  2. get rent contract
+  3. get rentable nodes
+  4. unreserve nodes
+
+ ** HINT: Make sure to modify it according to your needs and ensure to follow coding best practices. 
+ ** HINT: If you wish to unrent a node that you've just rented, modify the 'unreserve' caller to have the rented node's ID hard-coded and comment out the rest of the script to prevent re-renting another node.
+*/
+
+// Import required modules
+import { GridClient } from "../src/client";
 import { getClient } from "./client_loader";
 import { log } from "./utils";
 
+// Main function
 async function main() {
-  const client = await getClient();
+  try {
+    // Initialize the client
+    const client = await getClient();
 
-  async function unreserve() {
-    const unreserved = await client.nodes.unreserve({ nodeId: 12 });
-    log(unreserved);
-  }
+    // Get a list of rentable nodes
+    const rentableNodes = await getRentableNodes(client);
```

Comment:
in case there are no nodes to rent, the script shouldn't exit silently

Reply:
Please take a look again.